### DISCUSSION TITLE
Start blogging

### DISCUSSION THREAD

**Discussion by giscus** [Emilien-Foissotte/emilien-foissotte.github.io#3] [Articles]
# Start blogging

Introduction Hi, I will try to kick off some content on this blog, in order to hold details about some fun stuff I actually enjoyed making, or that might help other doing so.
To start with some technical details here, I think that explaining how this website is generated is a good start.
Making this kind of website This website is made public using the fancy tool of Github Pages and the behind the scenes are curated by Hugo.

http://localhost:1313/posts/2023/02/bootstrapping-website/

**Comment by Emilien-Foissotte** [comment]
Hope it works !

**Reply by Emilien-Foissotte** [reply]
Seems to be good 🎉
